### PR TITLE
Linting fixes - reduce complexity, resolve duplicative test names

### PR DESCRIPTION
**Pull request by rachelschneiderman**

### CODE REVIEW

File: shared/src/business/utilities/getFormattedCaseDetail.test.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ const {
 } = require('./getFormattedCaseDetail');
 const { calculateISODate, createISODateString } = require('./DateHandler');
 const { MOCK_USERS } = require('../../test/mockUsers');
-
+//
```
Comment (rachaelparris) [inline shared/src/business/utilities/getFormattedCaseDetail.test.js]: Can be removed

File: web-client/src/views/StartCase/Address.jsx [hunk 2]
```diff
@@ -27,6 +27,142 @@ export const Address = connect(
...
+        <Mobile>
+          <FormGroup
+            errorText={
+              validationErrors &&
+              validationErrors[type] &&
+              validationErrors[type].city
+            }
+          >
+            <label className="usa-label" htmlFor={`${type}.city`}>
+              Mobile City
```
Comment (rachaelparris) [inline web-client/src/views/StartCase/Address.jsx]: Was this left in by mistake?

**Review by rachaelparris — APPROVED** [review]

File: shared/src/business/useCases/migrateCaseInteractor.js [hunk 3]
```diff
@@ -67,6 +67,133 @@ const createUserAccount = async ({
 
 exports.createUserAccount = createUserAccount;
 
+const deleteExistingCase = async ({ applicationContext, docketNumber }) => {
```
Comment (sutt0n) [inline shared/src/business/useCases/migrateCaseInteractor.js:70]: I think as these get larger, we might want to consider extracting them out as `useCaseHelper`s so we can easily test them separately.

**Review by sutt0n — APPROVED** [review]

**Review by kkoskelin — APPROVED** [review]
👍

**Pull request merged**